config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
   root to: "givers#new"
-  resources :givers, only: [:new, :create]
+  resources :givers, only: [:new, :create, :show]
+  resources :access_accounts, only: [:new, :create, :show]
+  get '/token_sessions/:token' => 'token_sessions#create', as: :token_sessions
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.